assets/js/blocks/cart-checkout/mini-cart/style.scss

```
@@ -1,18 +1,22 @@
-.wc-block-mini-cart {
+.wc-block-mini-cart.wp-block-woocommerce-mini-cart {
+	background-color: transparent !important;
 	display: flex;
 	justify-content: flex-end;
+
+	&.is-transparent .wc-block-mini-cart__button {
+		background: transparent !important;
```

Not a big deal, but I think using `background-color` would be better here? (It would keep it consistent with line 2):
```suggestion
		background-color: transparent !important;
```